{{ cookiecutter.project_slug }}/README.md

```diff
@@ -103,6 +103,30 @@ and uncomment following code:
 ```
 in `dist.py` settings
 
+#### Sentry
+First you should create new Sentry project. If you've done it already you have to
```

Comment:
I would like to see the link to (at least) the main page of Sentry so that the user can immediately access the service required to setup the production build.

> Side note: there is a [Sentry Terraform provider](https://github.com/jianyuan/terraform-provider-sentry). I think we might consider using it in the future to automate the setup of prodcution build.